src/configuration/contracts.ts

```diff
@@ -0,0 +1,38 @@
+interface ContractNetworkConfig {
```

Comment:
you could pull this information from https://github.com/gnosis/safe-deployments

Reply:
Much better :)

Reply:
TBD in a separate PR